test/acceptance/view_haggadah_test.clj

```diff
@@ -6,52 +6,57 @@
              [etaoin.keys :as k]))
 
 (t/use-fixtures :once c/init-firebase)
-(t/use-fixtures :each c/with-screenshot)
-(t/use-fixtures :each c/delete-fs-emulator-data)
+(t/use-fixtures :each c/with-screenshot c/delete-fs-emulator-data)
 
 (defn click-on-haggadah
   [d text]
   (doto d
-    (e/screenshot "screenshots/create-haggadah-test-admin-exists-before-clicking-haggadah.png")
     (e/click-visible {:data-testid :haggadah-link} {:timeout 15})
-    (e/screenshot "screenshots/edit-haggadah-test-viewing-edited-haggadah.png")
-    (e/wait-has-text-everywhere text {:timeout 15})
-    (e/screenshot "screenshots/create-haggadah-test-admin-exists-haggadah-text.png")))
+    (e/wait-has-text-everywhere text {:timeout 15})))
 
-(def haggadah-title
+(def expected-haggadah-title
   "haggadah2023")
 
 (def actual-haggadah-text
```

Comment:
Perhaps is worth putting this expected value (not _actual_) closer to when it is used... unless is truly used multiple times or something like that

Reply:
I'll define the title during the test so it is closer to where it is used.